test/functional/import-abort-rescan.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python3
+# Copyright (c) 2017 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Test wallet import RPCs.
+
+Test rescan behavior of importprivkey when aborted. The test ensures that:
+1. The abortrescan command indeed stops the rescan process.
+2. Subsequent rescan catches the aborted address UTXO
+"""
+
+from decimal import Decimal
+import threading # for bg importprivkey
+import time      # for sleep
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import (assert_equal, get_rpc_proxy)
+
+class ImportAbortRescanTest(BitcoinTestFramework):
```

Comment:
It seems you are not specifying the constructor (`__init__`). Thus 2 additional nodes are spun up and remain unused.